onbuild/r2d_overlay.py

```diff
@@ -46,10 +47,10 @@ def apply_environment():
 def apply_requirements():
     req_path = binder_path('requirements.txt')
     env_path = binder_path('environment.yml')
-
+    NB_PYTHON_PREFIX=os.environ['NB_PYTHON_PREFIX']
```

Comment:
nit: (here and above), can you add a space around the `=`?

Reply:
ok. added tests, including checking currently pinned versions - could be more sophisticated, but i think is good for now